compiler/src/iree/compiler/Codegen/Common/GPU/test/gpu_nested_layout_vector_distribution.mlir

```diff
@@ -1002,3 +1002,86 @@ builtin.module attributes { transform.with_named_sequence } {
 
 // CHECK:         %[[T7:.+]] = vector.transpose %[[RD7]], [1, 2, 0] : vector<4x1x2xf16> to vector<1x2x4xf16>
 // CHECK:         vector.transfer_write %[[T7]], %arg0[%[[DIM2_ID4]], %[[DIM2_ID3]], %[[DIM0_ID]]]
+
+// -----
+
+#nested = #iree_vector_ext.nested_layout<
+  subgroups_per_workgroup = [1, 1], 
+  batches_per_subgroup = [2, 2], 
+  outers_per_batch = [1, 1], 
+  threads_per_outer = [16, 4], 
+  elements_per_thread = [1, 4], 
+  subgroup_order = [1, 0], 
+  batch_order = [1, 0], 
+  outer_order = [1, 0], 
+  thread_order = [1, 0], 
+  subgroup_basis = [1, 1], 
+  thread_basis = [4, 16]
```

Comment:
Can you add a comment explaining what is the shuffle width and how many elements we reduce per thread?

Reply:
added